packages/SwingSet/src/types-external.js

```diff
@@ -2,6 +2,11 @@
 
 export {};
 
+/**
+ * @import {Guarded} from '@endo/exo')
+ * @import {Passable, RemotableObject} from '@endo/pass-style')
```

Comment:
stray `)`s:

```suggestion
 * @import {Guarded} from '@endo/exo';
 * @import {Passable, RemotableObject} from '@endo/pass-style';
```

Reply:
thanks. I grepped for more and cleaned them up